test/system/libs/helpers.bash

```diff
@@ -27,6 +30,20 @@ function cleanup_containers() {
 }
 
 
+function setup_containers_store() {
+  mkdir -p ${TEMP_STORAGE_DIR}
+  # Setup a storage config file for PODMAN
+  echo -e "[storage]\n  driver = \"overlay\"\n  rootless_storage_path = \"${ROOTLESS_PODMAN_STORE_DIR}\"\n" > ${PODMAN_STORE_CONFIG_FILE}
```

Comment:
Question: `ROOTLESS_PODMAN_STORE_DIR` will be created by Podman or does it have to be created manually?

Reply:
Seems like it is created by Podman itself.